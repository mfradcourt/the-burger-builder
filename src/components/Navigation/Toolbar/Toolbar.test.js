import React from 'react';
import { shallow } from "enzyme";

import Toolbar from './Toolbar';

describe('<Toolbar />', () => {
  let wrapper;

  it('should render Toolbar', () => {
    wrapper = shallow(
      <Toolbar/>
    );

    expect(wrapper).toMatchSnapshot();
  });
});
