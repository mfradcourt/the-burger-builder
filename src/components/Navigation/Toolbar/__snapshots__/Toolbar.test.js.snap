// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Toolbar /> should render Toolbar 1`] = `
<header
  className="Toolbar"
>
  <drawerToggle />
  <div
    className="Logo"
  >
    <logo />
  </div>
  <nav
    className="DesktopOnly"
  >
    <navigationItems />
  </nav>
</header>
`;
